VideoWeb/VideoWeb/ClientApp/src/app/security/auth-base.guard.spec.ts

```diff
@@ -57,24 +51,19 @@ describe('authguard', () => {
     });
 
     describe('when login failed with unsuccessful authentication', () => {
-        const testcases = [
-            { flag: true, routePath: `/${pageUrls.IdpSelection}` },
-            { flag: false, routePath: `/${pageUrls.Login}` }
-        ];
+        const testcases = [{ flag: true, routePath: `/${pageUrls.IdpSelection}` }];
         testcases.forEach(test => {
```

Comment:
this doesn't need to be test cases since the feature flag is now removed and there's only one route path expected

Reply:
test removed